src/ray/protobuf/common.proto

```diff
@@ -131,13 +131,22 @@ message RayException {
   string formatted_exception_string = 3;
 }
 
+message ContainerOption {
+  /// The worker process container image
+  string image = 1;
+  /// The worker process container run options
+  repeated string run_options = 2;
+}
+
```

Comment:
I think it's not necessary to add this in protobuf. This setup probably will go to shimprocess, and in shimprocess, we don't do decode in raylet. cc @architkulkarni

Reply:
Makes sense, we should be able to do the setup in Python in `setup_runtime_env.py`.